torch/csrc/jit/runtime/static/ops.cpp

```diff
@@ -98,7 +104,14 @@ getOutOfPlaceOperation(Node* n) {
       auto out_t = p_node->Output(0, reg).toTensor();
       at::native::sigmoid_out(out_t, in0_t);
     };
-  } else if (n->kind() == c10::Symbol::fromQualString("aten::transpose")) {
+  }
+
+  return [](const ProcessedNode*, std::vector<IValue>&) { TORCH_CHECK(0); };
+}
+
+std::function<void(const ProcessedNode*, std::vector<IValue>&)>
+getNativeOperation(Node* n) {
```

Comment:
are these all "view producing" operators? add a comment/todo to extend them in the future (I think there are about 10 of those if I recall correctly: e.g. view, narrow, etc - we must have a list somewhere in jit already)

Reply:
Yes, in most of the cases they return a Tensor view, though it can return a copy of `self` if they're the same:

https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/TensorShape.cpp#L1617-L1619

Another corner case is when the input tensor is mkldnn. Static Runtime does not support this case yet.